kentik_api_library/kentik_api/synthetics/synth_tests/base.py

```diff
@@ -290,6 +291,46 @@ class SynTest(_ConfigElement):
     _created_by: UserInfo = field(default_factory=UserInfo, init=False)
     _last_updated_by: UserInfo = field(default_factory=UserInfo, init=False)
 
+    def _common_attribute_fixups(self):
+        # Correct settings.health_settings.activation parameters if necessary
+        orig = None
+        if not self.settings.health_settings.activation.times:
+            self.settings.health_settings.activation.times = "3"
+        min_activation_window = int(
+            self.settings.period * (int(self.settings.health_settings.activation.times) + 1) / 60
```

Comment:
Hmm maybe we should store the activation.times and activation.grace_period as int?

Reply:
and time_window as well

Reply:
The API unfortunately expects strings (I don't like it, but it is not an easy thing to change). Would the serialization to/from protobuf do the conversion?